docs/sysadmin-basics/cluster-config.txt

```diff
@@ -229,13 +229,14 @@ The master supports the following configuration settings:
    -  ``image``: Defines the default docker image to use when executing the workload. If a docker
       image is specified in the :ref:`experiment config <exp-environment-image>` this default is
       overriden. This image must be accessible via ``docker pull`` to every Determined agent machine
-      in the cluster. Users can use different container images for GPU vs. CPU agents differently by
-      specifying a dict with two keys, ``cpu`` and ``gpu``. Default values:
+      in the cluster. Users can configure different container images for NVIDIA GPU tasks using
+      ``cuda`` key (``gpu`` prior to 0.17.6), CPU tasks using ``cpu`` key, and ROCm (AMD GPU) tasks
```

Comment:
non-blocking: I think we normally don't embed historical docs into our present docs, so IMO you would be fine simply saying `cuda` here and ignoring `gpu`.  But I also don't think it hurts to write it how you have it.

Reply:
yep, I believe it'd be helpful reference for existing users who'd be searching for `gpu` to see that information inline.